core/domain/user_query_jobs_one_off.py

```diff
@@ -139,7 +178,8 @@ def map(user_settings_model):
 
         predicates = constants.EMAIL_DASHBOARD_PREDICATE_DEFINITION
         for predicate in predicates:
-            if getattr(query_model, predicate['backend_attr']) is not None:
+            value = getattr(query_model, predicate['backend_attr'])
+            if value is not None and value is not 'None':
```

Comment:
Why `'None'` as a string? Could we use `None`?

Reply:
Changed.